src/gameplay/game.ts

```diff
@@ -49,12 +52,16 @@ class Game extends Room {
       maxNumPlayers_,
       newRoomPassword_,
       gameMode_,
-      // gameType,
       ranked_,
     );
 
     this.callback = callback_;
 
+    this.roomCreationType = roomCreationType_ === "RANKED_QUEUE" ? roomCreationTypeEnum.RANKED_QUEUE : 
+        roomCreationType_ === "UNRANKED_QUEUE" ? roomCreationTypeEnum.UNRANKED_QUEUE : 
+        roomCreationType_ === "CUSTOM_ROOM" ? roomCreationTypeEnum.CUSTOM_ROOM : 
+        roomCreationTypeEnum.INVALID_ROOM;
```

Comment:
Let's just throw if there isn't a match.
Let's also make a function to parse the string into the enum.